src/pyedb/dotnet/edb_core/stackup.py

```diff
@@ -68,16 +68,161 @@
 logger = logging.getLogger(__name__)
 
 
-class Stackup(object):
+class LayerCollection(object):
+    AUTO_REFRESH = True
```

Comment:
This will create a class parameter and changing the value in one instance will impact other instances. I'm not sure that this is the behavior that you would like to have. If that's the case, please forget the previous comment.

Reply:
@SMoraisAnsys yes, this is intended. AUTO_REFRESH is a global parameter. With AUTO_REFRESH==False, LayerCollection is saved aside before all layer changes are made.